packages/contractkit/CHANGELOG.md

```diff
@@ -17,6 +17,10 @@ Other Changes
 
 ## Published
 
+### **[0.4.18]** -- 2020-10-30
+Features
+- ODIS Client Update - [#5621](https://github.com/celo-org/celo-monorepo/pull/5621)]
+
```

Comment:
I think #5270, #5091, #5624, & #5436 also need to be added here

Reply:
Added, thanks! Also added a comment to the Notion doc to do this check:
https://www.notion.so/DevX-Package-Release-Guide-8da831a90dc8414fab93653f0228057b